docs/design/devices.md

```diff
@@ -0,0 +1,34 @@
+# K8s support "/sys/fs/cgroup/devices"
+This document suggest a way for support hardware which could under "/sys/fs/cgroup/devices". So far, we could try to add GPU as a sample, then extend the others.
+
+## Background
+GPU is important to lots of workloads, such as machine learning, video stream decoding/coding, etc... But Kubernetes doesn't support schedule/tracking it which need to be.
+Also, we can think to add all the devices could be assigned to container to k8s, it could make some hardware more efficient.
```

Comment:
This time I only want to enable GPU in kubelet and change the interface of scheduler to could add more devices in the future.